### ISSUE TITLE
ReferenceErrors not displayed correctly in case of Promises.

### ISSUE THREAD

**Issue by vinaygw** [lukehaas/RunJS#196]
var r = new Promise(function(resolve,reject){
  resolve(10)
}).then(x=>conosole.log(x));

No reference error for (conosole) show up in the output.

**Comment by lukehaas** [comment]
@vinaygw thanks for raising this. I will investigate.

**Comment by lukehaas** [comment]
This will be fixed in the next release.

**Comment by Victor-Ugwueze** [comment]
Just out of curiosity, is there a keyword `conosole` ?
I think it should be spelt `console`

```js
var r = new Promise(function(resolve,reject){
resolve(10)
}).then(x=>console.log(x));

```

**Comment by lukehaas** [comment]
@Victor-Ugwueze exactly, `conosole` is not defined so an error should be thrown when calling it.

**Comment by lukehaas** [comment]
@vinaygw this is now resolved in version 1.13.0
